helix-term/src/commands.rs

```diff
@@ -1019,6 +1019,34 @@ mod cmd {
         doc.format(view.id)
     }
 
+    fn earlier(editor: &mut Editor, args: &[&str], event: PromptEvent) {
+        let sotp = match args.join(" ").parse::<helix_core::StepsOrTimePeriod>() {
```

Comment:
I think to make things simpler we can just follow vim.
```
:earlier {count}        Go to older text state {count} times.
:earlier {N}s           Go to older text state about {N} seconds before.
:earlier {N}m           Go to older text state about {N} minutes before.
:earlier {N}h           Go to older text state about {N} hours before.
:earlier {N}d           Go to older text state about {N} days before.
:earlier {N}f           Go to older text state {N} file writes before. // this not necessary in this PR
```

Reply:
That's roughly what I had in mind. We could be a bit more clever than vim, for example we could try to support "1h 30m" or "1 hour 30 minutes".

Reply:
I don't think it's good to have space in it, because it makes stuff harder. But yeah `1h30m` is good enough at least for me.

Reply:
We have a similar functionality now with the time spans.